src/octree/mod.rs

```diff
@@ -224,7 +217,11 @@ impl Octree {
     }
 
     pub fn get_visible_nodes(&self, projection_matrix: &Matrix4<f64>) -> Vec<NodeId> {
-        let frustum = collision::Frustum::from_matrix4(*projection_matrix).unwrap();
+        let frustum =
```

Comment:
This is interesting: We now use our own `Frustum`'s intersection testing. Axes are precomputed.